packages/web-components/fast-components/src/text-area/text-area.ts

```diff
@@ -56,25 +56,17 @@ export class TextArea extends FormAssociated<HTMLTextAreaElement> {
 
     public textarea: HTMLTextAreaElement;
 
-    @attr
+    @attr({ mode: "boolean" })
     public autofocus: boolean;
     private autofocusChanged(): void {
         if (this.proxy instanceof HTMLElement) {
             this.proxy.autofocus = this.autofocus;
         }
     }
 
-    @attr
+    @attr({ converter: nullableNumberConverter, mode: "fromView" })
```

Comment:
are these not needed on textfield as well?

Reply:
fixed.